src/rts/GameState.java

```diff
@@ -38,6 +39,9 @@
     protected HashMap<Unit,UnitActionAssignment> unitActions = new LinkedHashMap<>();
     protected UnitTypeTable utt;
 
+    protected int [][][][] matrixObservation;
+    public static final int numFeatureMaps = 5;
```

Comment:
Maybe all the JNI-related variables and functions could have some "JNI" prefix? like "JNImatrixObservation", "JNInumFeatureMaps", etc.?

Reply:
Hey Santi thanks for the review. I feel this `matrixObservation` could also be used in socket stuff, which is the reason I have put it without the `JNI`-prefix.